Modules/DataModel/Tests/PocketCastsDataModelTests/AutoAddCandidatesDataManagerTests.swift

```diff
@@ -0,0 +1,93 @@
+import XCTest
+import FMDB
+import PocketCastsUtils
+import SQLite3
+@testable import PocketCastsDataModel
+
+final class AutoAddCandidatesDataManagerTests: XCTestCase {
+
+    private var overriddenFlags = [FeatureFlag: Bool]()
+
+    private func override(flag: FeatureFlag, value: Bool) throws {
+        overriddenFlags[flag] = flag.enabled
```

Comment:
There's a `FeatureFlagMock` we can use that does that. I think we can use this among the different unit tests.